ktor-client/ktor-client-features/ktor-client-logging/common/src/io/ktor/client/features/logging/Logging.kt

```diff
@@ -50,19 +50,14 @@ class Logging(
         if (level.body) logRequestBody(content)
     }
 
-    private suspend fun logResponse(response: HttpResponse): Unit = response.use {
+    private suspend fun logResponse(response: HttpResponse) {
         if (level.info) {
             logger.log("RESPONSE: ${response.status}")
             logger.log("METHOD: ${response.call.request.method}")
             logger.log("FROM: ${response.call.request.url}")
         }
 
         if (level.headers) logHeaders(response.headers.entries())
-        if (level.body) {
```

Comment:
So who does `logResponseBody` instead of this?

Reply:
In the feature itsel :168